src/ncp/ncp_base.cpp

```diff
@@ -42,6 +42,8 @@ namespace Thread {
 
 extern ThreadNetif *sThreadNetif;
 
+static NcpBase *sNcpContext = NULL;
```

Comment:
This is fine for now (would rather not overload this pull request with callback refactoring), but we may want to consider changing the callback pattern to something that wouldn't require stuff like this.